README.md

```diff
@@ -15,6 +15,10 @@ This is a public, community-contributed repository that contains:
 
 [Az.DevTestLabs](https://github.com/Azure/azure-devtestlab/tree/master/samples/DevTestLabs/Modules/Library), for short, is a PowerShell module to simplify the management of [Azure DevTest Labs](https://azure.microsoft.com/en-us/services/devtest-lab/). It provides composable functions to create, query, update and delete labs, VMs, Custom Images and Environments. The source can be located under [/samples/DevTestLabs/Modules/Library/](https://github.com/Azure/azure-devtestlab/tree/master/samples/DevTestLabs/Modules/Library).
 
+**PowerShell Module Status:**
```

Comment:
Not sure this is required here. You already have it in the library `README.md`, to which the above text already links.

Reply:
No problem - @leovms , I can get rid of the status here now.

Reply:
@leovms - done!  I got rid of this from the main readme.